index.bs

```diff
@@ -24,6 +24,68 @@ Assume Explicit For: yes
 Blah blah blah.
 </section>
 
+<h2 id="deployed-mitigations">
+Some browsers have deployed and announced protections against
+navigational-tracking.  This section is a work in progress to detail what
+protections have been shipped and / or are planned. This section should not
+be understood to be comprehensive.
+
+<h3>Safari</h3>
+Safari uses a heuristic-based approach to combat navigational-tracking. Safari uses
+a client-side machine classifier to determine which sites (determined by
+eTLD+1) are participating in navigational-tracking. The main inputs to this
```

Comment:
"… are potentially participating …" since there is no way to know client-side what's being done server-side.